### ISSUE TITLE
extension not working in code-server

### ISSUE THREAD

**Issue by einsitang** [eclipse-oct/open-collaboration-tools#84]
I am use [microsoft/vscode](https://github.com/microsoft/vscode) run code-server with open-collaboration-tools extension on my server for internal service,use browser vscode collaboration online live code review.

after `Join Collaboration Session`, brwoser with reflush and not sync workspace.

but `vscode.dev` and other web IDE(github codespace himself) is work fine . 

I'am running on my macos and github codespace is not working too

I don't know whether this issue comes from `code-server`(microsoft/vscode) or the current extension. I hope you can help check it if you are free

**Comment by msujew** [comment]
I actually haven't tested the code-server environment yet. Unfortunately, every vscode environment (i.e. desktop, web, server) seems to behave differently when it comes to the specific behavior of loading a virtual workspace. I'll take a look at this once I'm back from vacation early next year. Note that it seems to work correctly on code-server from coder, see https://github.com/eclipse-oct/open-collaboration-tools/issues/75.

**Comment by einsitang** [comment]
here is my code-server run step:

# clone source compile and run 
git clone https://github.com/microsoft/vscode.git
cd vscode
npm i
npm run compile
./script/code-server.sh --lanuch

# add extension gallery config to product.json
```
	"extensionsGallery": {
		"serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
		"cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
		"itemUrl": "https://marketplace.visualstudio.com/items"
	}
```
# install oct extension in code-server(online vscode) with other terminal
./script/code-server.sh --install-extension typefox.open-collaboration-tools

**Comment by einsitang** [comment]
> I actually haven't tested the code-server environment yet. Unfortunately, every vscode environment (i.e. desktop, web, server) seems to behave differently when it comes to the specific behavior of loading a virtual workspace. I'll take a look at this once I'm back from vacation early next year. Note that it seems to work correctly on code-server from coder, see #75.

yes . `vscode.dev` and other vscode WEB IDE is work fine . but not work for my self-build . It's very strange but I dont know why

**Comment by einsitang** [comment]
hello，something news？

**Comment by msujew** [comment]
Unfortunately no meaningful progress, since I (and the rest of the team) have very little bandwidth for such issues right now. Feel free to debug the issue and post results in here or open a PR to fix the issue.

**Comment by einsitang** [comment]
OK, thank you for your reply. I'll try to figure out the reason for this issue. If I make any progress, I'll inform you.

**Comment by msujew** [comment]
I believe https://github.com/eclipse-oct/open-collaboration-tools/pull/150 should fix the issue. Sorry for the slow reply on this.

**Comment by einsitang** [comment]
> I was able to confirm that the fix for this works as expected in GitHub Codespaces as of extension version 0.3.4 :)

Thank you for your attention and for solving this issue. I will test it when I have some free time. If there are any issues, I will inform you again. Thank you very much.
